### PR TITLE
OCPBUGS#43373: Updated the ROSA docs to explain SDN support

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.16 and 4.17

Issue:
[OCPBUGS-43373](https://issues.redhat.com/browse/OCPBUGS-43373)

Link to docs preview:
* [Dedicated](https://83557--ocpdocs-pr.netlify.app/openshift-dedicated/latest/networking/about-managed-networking.html)
* [ROSA](https://83557--ocpdocs-pr.netlify.app/openshift-rosa/latest/networking/about-managed-networking.html)

- [x] SME has approved this change.
- [x] QE has approved this change.

Additional resources
* [Slack thread](https://redhat-internal.slack.com/archives/C0308P3FC7R/p1729672973951729)

### CODE REVIEW

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Comment by zhaozhanqi** [comment]
LGTM

**Review by jneczypor — COMMENTED** [review]
One small note. Otherwise, LGTM
/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: _topic_maps/_topic_map_rosa.yml [hunk 1]
```diff
@@ -1130,10 +1130,15 @@ Topics:
 - Name: OVN-Kubernetes network plugin
   Dir: ovn_kubernetes_network_provider
   Topics:
-  - Name: About the OpenShift SDN network plugin
+  - Name: About the OVN-Kubernetes network plugin
     File: about-ovn-kubernetes
   - Name: Configuring an egress IP address
     File: configuring-egress-ips-ovn
+- Name: OpenShift SDN default CNI network provider
```
Comment (laubai) [inline _topic_maps/_topic_map_rosa.yml:1137]: This can just be "OpenShift SDN network plugin"

**Pull request merged**